requirements_test.txt

```
@@ -24,6 +24,6 @@ sphinx_design<0.6.0
 sympy<1.13.0
 tqdm<4.67.0
 trame>=2.5.2,<3.6
-trame-vtk>=2.5.8,<2.9
+trame-vtk>=2.5.8,<2.8.4
```

Let's add a comment so that we can refer to it when we drop support for Python 3.8.
```suggestion
trame-vtk>=2.5.8,<2.8.4 # 2.8.4 break compatibility with Python3.8
```